docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,146 @@
-* xref:getting-started:hello-world-cli.adoc[]: How to execute queries from the command line using the Kaskada CLI. 
-This quick tutorial will help you install and run Kaskada from the command line, load some sample data into Kaskada, and use Kaskada queries to explore that data all on the command line.
+[TIP]
+====
+Before jumping in to writing queries in Kaskada, it's a good idea to take a minute understanding "how to think with Kaskada".
+Kaskada is built on timelines - a simple, powerful abstraction that may be different than what you're accustomed to.
+
+If you'd prefer to jump right in, scroll down to the xref:#quick-starts[] section
+====
+
+== Thinking with Kaskada's timelines
+
+Kaskada is built on the idea of a _timeline_ - the history of how a value changes over time for a specific entity or group.
+
+[stream_viz,name=basic-sum]
+.Aggregating events as a timeline
+....
+[
+    {
+        "label": "Purchase.amount",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "2"},
+                {"t": 13, "v": "13"},
+                {"t": 18, "v": "4"}
+            ]
+        ]
+    },
+    {
+        "label": "Purchase.amount | sum()",
+        "kind": "continuous",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "7"},
+                {"t": 13, "v": "20"},
+                {"t": 18, "v": "24"}
+            ]
+        ]
+    }
+]
+....
+
+Timelines allow you to reason about temporal context, time travel, sequencing, time-series, and more. 
+They allow simple, composable, declarative queries over events. 
+Transforming and combining timelines allows you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation's inputs and outputs are timelines. 
+The timeline is a flexible abstraction that can be used in different ways depending on your needs.
+In some cases, you may want to know what the timeline's value is at a specific point in time, in other cases you may want to know how the timeline's value changes over time.
+
+When making a query, you configure how to use the query's output timeline: you can use the timeline as either a _history_ or as a _snapshot_.
+
+* A timeline *History* contains a value each time the timeline changes, and each row describes a different entity at a different point in time. 
+* A timeline *Snapshot* contains a value for each entity at the _same_ point in time; each row is associated with a different entity, but all rows reflect the same point in time.
+
+This output may be written in different ways -- for example, they may be written to a Parquet file or sent as events to a stream.
```

Comment:
`This output` -> `Both history and snapshot outputs`

(specifically, clarify that we're not binding to the most recent output -- snapshot -- but both kinds)